src/Repository/AliasRepository.php

```diff
@@ -39,4 +39,14 @@ public function findByUser(User $user)
     {
         return $this->findBy(['user' => $user]);
     }
+
+    public function findRandomByUser(User $user)
+    {
+        return $this->findBy(['user' => $user, 'random' => true]);
+    }
+
+    public function findCustomByUser(User $user)
+    {
+        return $this->findBy(['user' => $user, 'random' => false]);
+    }
```

Comment:
Do prevent code duplication, we could instead add an optional parameter `$random` to `findTypeByUser` and pass it to `findBy`?

```
    public function findTypeByUser(User $user, bool $random = false)
    {
        return $this->findBy(['user' => $user, 'random' => $random]);
    }
```